contracts/governance/Governor.sol

```diff
@@ -0,0 +1,110 @@
+pragma solidity ^0.5.11;
```

Comment:
Note to self: fix compiler version, this is considered good practice. Not really super important for our use case but if someone compiles this contract with a different compiler version above 0.5.11 installed there could be differences in behaviour.